SPECS/libtomcrypt/libtomcrypt.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        A comprehensive, portable cryptographic toolkit
 Name:           libtomcrypt
 Version:        1.18.2
-Release:        8%{?dist}
+Release:        9%{?dist}
```

Comment:
I noticed this just now - it looks like there's a `BuildRequires: libtommath-devel`, which is an extended package. I'd double-check that this one build in the presence of **ONLY** core packages. If not, you will need to move the dependencies from extended specs as well.

Short explanation:

Core packages cannot have any external dependencies and so must build by only relying on themselves. Extended packages are allowed to use core and extended, so each extended -> core migration must take into account the extended-only dependencies.